### PR TITLE
Assignment7

### PR DESCRIPTION
**Pull request by twofingerrightclick**
I think I did the rebase right!

### CODE REVIEW

File: DelegateFun/Sorter.Tests/SortUtilityTests.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public bool IsGreater(int first, int second)
 
         public bool IsLess(int first, int second)
         {
-            return first > second;
+            return first < second;
         }
```
Comment (ashermancinelli) [inline DelegateFun/Sorter.Tests/SortUtilityTests.cs]: Is this not implicit in the `<` operator?

File: DelegateFun/Sorter.Tests/SortUtilityTests.cs [hunk 2]
```diff
@@ -14,41 +13,82 @@ public class SortUtilityTests
...
+        }
+
+
+        [TestMethod]
+        public void SortUtility_ShouldSortDescending_UsingLambdaStatement()
+        {
+            //setup
+            int[] array = arrayAscending;
+
+            Func< int, int, bool> sortByDescending = (x, y) => { return x > y; };
```
Comment (ashermancinelli) [inline DelegateFun/Sorter.Tests/SortUtilityTests.cs]: Just pointing out that the readme says not to use `Func`

Comment (twofingerrightclick) [reply DelegateFun/Sorter.Tests/SortUtilityTests.cs]: Thanks! I realize now I can implicitly cast to the defined delegate.

**Comment by skozubenko1** [comment]
I like how you break everything down into sections. I found it helpful in understanding the assignment when I was working on it. Thank you!

File: DelegateFun/Sorter/SortUtility.cs [hunk 3]
```diff
@@ -1,10 +1,81 @@
...
+            SortBy = sortBy;
+        }
         // Sort method should be implemented here
         // It should accept an int[] and a delegate you define that performs the actual comparison
+
+        /* low  --> Starting index,  high  --> Ending index */
+
+        public void Sort(int[] arr)
+        {
+            if (arr is null)
```
Comment (breaman) [inline DelegateFun/Sorter/SortUtility.cs]: Where is the unit test around this check?

**Pull request closed without merging**